pkg/server/devfile-handler.go

```diff
@@ -22,6 +23,35 @@ import (
 	"github.com/devfile/library/pkg/devfile/parser/data/v2/common"
 )
 
+func (s *Server) devfileSamplesHandler(w http.ResponseWriter, r *http.Request) {
+
+	var data devfileSamplesForm
+	registry := "pkg/devfile/sample-placeholder.json"
+
+	err := json.NewDecoder(r.Body).Decode(&data)
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to decode response: %v", err)
+		klog.Error(errMsg)
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: errMsg})
+		return
+	}
+
+	if data.Registry != "" {
+		registry = data.Registry
```

Comment:
Does this meant that all the registries will need to be part of the backend code ?

Reply:
Devfile registries will be in https://github.com/devfile/registry or anywhere else where its decided to be hosted. The index placeholder json here is what will be returned, when the sample is in place at https://github.com/devfile/registry